src/_gettsim/interface.py

```diff
@@ -373,6 +359,57 @@ def _create_input_data(
     return input_data
 
 
+def _warn_if_functions_and_columns_overlap(
+    columns_overriding_functions: set[str],
+) -> None:
+    """Warn if functions which compute columns overlap with existing columns.
+
+    Parameters
+    ----------
+    columns_overriding_functions : set[str]
+        Names of columns in the data that override hard-coded functions.
+    """
+    if columns_overriding_functions:
+        n_cols = len(columns_overriding_functions)
+        first_part = format_errors_and_warnings(
+            f"Your data provides the column{'' if n_cols == 1 else 's'}:"
+        )
+        formatted = format_list_linewise(list(columns_overriding_functions))
+        second_part = format_errors_and_warnings(
+            f"""
+            {'This is' if n_cols == 1 else 'These are'} already present among the
+            hard-coded functions of the taxes and transfers system.
+            If you want {'this' if n_cols == 1 else 'a'} data column to be used
+            instead of calculating it within GETTSIM you need not do anything.
+            If you want {'this' if n_cols == 1 else 'a'} data column to be calculated
+            by hard-coded functions, remove {'it' if n_cols == 1 else 'them'} from the
+            *data* you pass to GETTSIM. {'' if n_cols == 1 else '''You need to pick one
+            option for each column that appears in the list above.'''}
+            """
+        )
+        how_to_ignore = format_errors_and_warnings(
+            """
+            If you want to ignore this warning, add the following code to your script
+            before calling GETTSIM:
+
+                import warnings
```

Comment:
```suggestion
                import warnings
                from gettsim import FunctionsAndColumnsOverlapWarning
```

Reply:
I guess this is a thing to put into the main namespace as many people will want to use it?